api/package.json

```diff
@@ -87,6 +87,7 @@
         "@types/swagger-ui-dist": "^3.30.3",
         "@types/uuid": "9.0.7",
         "chai": "^4.3.10",
+        "copyfiles": "^2.4.1",
```

Comment:
Should be a dev dependancy?

Reply:
It is, I believe, GitHub just collapsed the changes so it doesn't look like it.